go.mod

```diff
@@ -123,5 +122,6 @@ replace (
 	github.com/stretchr/testify => github.com/stretchr/testify v1.3.0
 	google.golang.org/appengine => google.golang.org/appengine v1.4.0
 	gopkg.in/yaml.v2 => gopkg.in/yaml.v2 v2.2.2
+	k8s.io/component-base => k8s.io/component-base v0.23.16
```

Comment:
Why replace?

Reply:
Because version 0.26 of `component-base` relies on a higher version of Prometheus, I'm concerned that upgrading will introduce unexpected errors. So I find replace it with a lower version.

Reply:
what if we directly set ```component-base``` to v0.23.16? would there be any issues?

Reply:
When I set to v0.23.16, every time I execute `go mod tidy`, it will upgrade to v0.26.12. So I have to force to replace it.

Reply:
I think upgrading Prometheus is fine. We have to do it sooner or later.